utils/oscap-podman

```diff
@@ -92,29 +93,26 @@ fi
 # podman init creates required files such as: /run/.containerenv - we don't care about output and exit code
 podman init $ID &> /dev/null || true
 
-DIR=$(podman mount $ID) || die "Failed to mount."
+DIR=$($ROOTLESS_CMD podman mount $ID) || die "Failed to mount."
 
 if [ ! -f "$DIR/run/.containerenv" ]; then
     # ubi8-init image does not create .containerenv when running podman init, but we need to make sure that the file is there
-    touch "$DIR/run/.containerenv"
+    $ROOTLESS_CMD touch "$DIR/run/.containerenv"
```

Comment:
I think the command used in the if condition (`[ ! -f "$DIR/run/.containerenv" ]`) also needs to be run with `podman unshare`.